api/models/Post.js

```diff
@@ -362,6 +362,26 @@ module.exports = bookshelf.Model.extend(Object.assign({
     PUBLIC_READABLE: 1
   },
 
+ projectsForContext: async function (context, contextId) {
+    let postIds
+    
+    switch (context) {
+      default: // For now, the only context is member profiles. Add to this once we show projects on group landing pages.
+        postIds = await this.query(q => {
+          q.select('posts.*')
```

Comment:
if you are only using the ids from this query then you can just select posts.id

Reply:
keeping it as * since it's the main query now